suite/tests/CMakeLists.txt

```diff
@@ -2333,11 +2333,15 @@ if (CLIENT_INTERFACE)
   endif ()
```

Comment:
nit: commit title: the text after the i# and before the colon is supposed to be a very short description of what the issue in general covers, with the text after the colon a description of what this particular commit does. 
 The before-colon text should match earlier commits, which say `i#2868 test failures: `.

Reply:
Done